modules/statics/src/networks.ts

```diff
@@ -430,6 +430,17 @@ class Ethereum2 extends Mainnet implements AccountNetwork {
   accountExplorerUrl = 'https://beaconscan.com/address';
 }
 
+class EthereumW extends Mainnet implements EthereumNetwork {
+  name = 'Ethereum PoW';
+  family = CoinFamily.ETHW;
+  explorerUrl = '';
+  accountExplorerUrl = '';
+  chainId = NaN; // This would be updated with the chain ID of a ETH PoW fork if it exists after the merge
```

Comment:
Can we add a TODO with ticket num https://bitgoinc.atlassian.net/browse/BG-55520 BG-55520?